plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -732,14 +732,7 @@ public void testTimestampColumnAndTimestampWithTimeZoneConstant()
     @Override
     public void testCaseSensitiveDataMapping(DataMappingTestSetup dataMappingTestSetup)
     {
-        try {
-            super.testCaseSensitiveDataMapping(dataMappingTestSetup);
-        }
-        catch (AssertionError ignored) {
-            // TODO https://github.com/trinodb/trino/issues/3645 - PostgreSQL has different collation than Trino
-            assertThatThrownBy(() -> super.testCaseSensitiveDataMapping(dataMappingTestSetup))
-                    .hasStackTraceContaining("not equal\nActual rows");
-        }
+        throw new SkipException("TODO - https://github.com/trinodb/trino/issues/3645 - PostgreSQL has different collation than Trino");
```

Comment:
add a `// TODO` comment with a link
otherwise my IDE doesn't highlight the TODO word